components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -129,15 +132,19 @@ func main() {
 
 	// create and run queue, steps provisioning
 	initialisation := provisioning.NewInitialisationStep(db.Operations(), db.Instances(), provisionerClient, directorClient, inputFactory, cfg.ManagementPlaneURL)
+
 	resolveCredentialsStep := provisioning.NewResolveCredentialsStep(db.Operations(), accountProvider)
+	provisionAzureEventHub := provisioning.NewProvisionAzureEventHubStep(db.Operations(), azure.NewAzureClient(), accountProvider, ctx)
 	runtimeStep := provisioning.NewCreateRuntimeStep(db.Operations(), db.Instances(), provisionerClient)
 	smOverrideStep := provisioning.NewServiceManagerOverridesStep(db.Operations(), cfg.ServiceManager)
 
 	logs := logrus.New()
 	stepManager := process.NewManager(db.Operations(), logs)
 	stepManager.InitStep(initialisation)
+
 	stepManager.AddStep(1, resolveCredentialsStep)
-	stepManager.AddStep(2, smOverrideStep)
+	stepManager.AddStep(2, provisionAzureEventHub)
+	stepManager.AddStep(3, smOverrideStep)
```

Comment:
Getting the SM overrides are not connected with azure hub, so they be done in parallel, am I right @jasiu001?

Reply:
in general, we do not support parallel steps yet but if you mean the same weight for step, you are right it could be the same weight (e.g. 2)

Reply:
So let's use the same weight, then we are future-proof if parallel steps feature is coming 😎